Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    before(:each) do
+      if described_class.instance_variable_defined?(:@svn)
```

Comment:
Here, I'm trying to remove the instance variable if already set because once `svn_available?` is called, it will set the `@svn` instance variable. To make these tests independent of each other, I need to ensure that the instance variable is not set beforehand. What do you suggest in this case?